Packs/PrismaCloud/Playbooks/playbook-PCR_-_GCP_Kub_Engine_Cluster_Misconfig.yml

```
@@ -0,0 +1,1070 @@
+id: Prisma Cloud Remediation - GCP Kubernetes Engine Cluster Misconfiguration
+version: -1
+fromversion: 5.0.0
+name: Prisma Cloud Remediation - GCP Kubernetes Engine Cluster Misconfiguration
+description: |-
+  This playbook remediates the following Prisma Cloud GCP Kubernetes Engine Cluster alerts.
+
+  Prisma Cloud policies remediated:
+
+  * GCP Kubernetes Engine Clusters Basic Authentication is set to Enabled
+  * GCP Kubernetes Engine Clusters have HTTP load balancing disabled
+  * GCP Kubernetes Engine Clusters have Legacy Authorization enabled
+  * GCP Kubernetes Engine Clusters have Master authorized networks disabled
+  * GCP Kubernetes Engine Clusters have Network policy disabled
+  * GCP Kubernetes Engine Clusters have Stackdriver Logging disabled
+  * GCP Kubernetes Engine Clusters have Stackdriver Monitoring disabled
+  * GCP Kubernetes Engine Clusters have binary authorization disabled
+  * GCP Kubernetes Engine Clusters web UI/Dashboard is set to Enabled
+  * GCP Kubernetes cluster intra-node visibility disabled
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 61bee172-14d4-4a48-815c-913b49bef800
+    type: start
+    task:
+      id: 61bee172-14d4-4a48-815c-913b49bef800
+      version: -1
+      name: ""
+      description: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "27"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 630,
+          "y": -430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: 1abd9220-e39e-4206-8aa3-dba2695c7f4e
+    type: title
+    task:
+      id: 1abd9220-e39e-4206-8aa3-dba2695c7f4e
+      version: -1
+      name: Done
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 630,
+          "y": 1180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 4b994469-67b1-4bcc-875b-a1b7922b49a4
+    type: regular
+    task:
+      id: 4b994469-67b1-4bcc-875b-a1b7922b49a4
+      version: -1
+      name: Get cluster details
+      description: Gets the details of a specific cluster.
+      script: GoogleKubernetesEngine|||gcloud-clusters-describe
```

```suggestion
      script: |||gcloud-clusters-describe
```
do not bind commands to specific instances

---

this is an integration cmd and again a result from the Demisto export.  do you still want the change?

---

changed.